### DISCUSSION TITLE
Github-Free private repo collaborator Limit for commercial use?

### DISCUSSION THREAD

**Discussion by sim1sang1songs1** [community/community#62379] [New to GitHub]
Hey there,
I am trying to select the correct license for a small team in my organization.

**What is the collaborator limit for a private repository for commercial use with the Github-Free plan?**

I remember seeing some articles saying it's limited to 3 users. 
Now I can't find them anymore but I can find Github sites stating that you can invite an unlimited amount of collaborators to a private repository but it doesn't specify whether or not this still counts for commercial use.

It would be very helpful if someone could clarify this preferably with an official source.

Thanks for any answers.

**Comment by queenofcorgis** [comment]
Hi there - I'm checking on this and waiting for the right answer - thanks for asking this!

**Comment by Livia-Rasp** [comment]
You can find the answer here: https://github.com/pricing under collaboration:

The number of collaborators on a free, private repo is unlimited. Though I think that in the past, it was limited.

**Comment by MahsaBakhtiari** [comment]
Hey there! When choosing a license for your project, it's essential to consider your goals and how you want others to use and contribute to your work. Some popular licenses include MIT (permissive) and GPL (copyleft). If you're unsure, seek legal advice or go with a permissive license like MIT to encourage broader adoption. About user limits, as of my last update, the GitHub Free plan limited private repositories to 3 collaborators for personal and commercial use. If you need more, consider upgrading to GitHub Pro for unlimited collaborators. Just remember, pricing and features can change, so best to double-check with GitHub directly. 😊
